docs/emoji/charts-and-data.md

```diff
@@ -1,32 +1,51 @@
-# Emoji Charts and Data
+# Posting Emoji Charts and Data
 
-## Emoji beta charts
+The following use 15.0 as the example version. Change that to whatever the version is.
 
-always check and post the files that have changed in:
+## Posting Emoji beta charts
 
-<https://www.unicode.org/draft/emoji/>
+Check and post the chart files that have changed in:
 
-*   \*.html
-*   charts-BBBB/\* (except /internal/) - where BBBB is the beta release version
-    — in this case 12.0
-*   [future/](https://www.unicode.org/draft/emoji/future/)\* (except /internal/)
-*   [images/](https://www.unicode.org/draft/emoji/images/)\*
+https://github.com/unicode-org/emoji/tree/main/docs/emoji/
 
-In this case you should also post the following (just regenerated) since there
-are new images, and some changes in layout.
+That is:
 
-*   charts-RRRR/\* (except /internal/) - where RRRR is the current release
-    version — in this case 11.0
+*   First sanity-check the charts.
+    *   You can do a link check on the "flagged" charts, since they are built to be small.
+    *   Visually inspect them also; 
+        *   make sure that the new characters are in, 
+        *   version numbers are right,
+        *   and images are present (at least one per row)
+*   https://github.com/unicode-org/emoji/tree/main/docs/emoji/charts-15.0
+    *   All files with .html
+    *   Skip other files, and /internal/
+*   https://github.com/unicode-org/emoji/tree/main/docs/emoji/future
+    *   All files with .html
 
-(linking checking to be done with the "flag" versions. please let me know of any
-problems (though if there are no visible artifacts it isn't a showstopper))
+NOTE: make sure that all the emoji images are in a proper directory with proper names:
+*  **not** proposed/proposed_...png, but rather
+*  sample/sample_...png, or
+*  android/androic_...png, or
+*  ...
 
+[Not sure this comment is still relevant]
 First time posting a new set of beta charts, make sure that charts-beta is a
 newly created symlink to charts-BBBB.
 
-## Emoji beta data
-
-Copy all files under draft/Public/emoji/NN.N files to live Public/emoji/NN.NN
-*   don't copy anything under the "internal" directory.
-
-Check the readme file if it exists, which should say draft.
+## Posting Emoji beta data
+
+Copy all files under https://github.com/unicode-org/unicodetools/tree/main/unicodetools/data/emoji/15.0 files to Public
+*   First sanity-check the files.
+    *   The Readme.txt should say "draft" in the contents, and have the version 15.0.
+    *   The other files should have the new 15.0 characters and sequences.
+    *   The previous characters should be unchanged, except that the CLDR names may change.
+*   Don't copy anything under the /internal/ directory.
+*   Copy the following to https://unicode.org/Public/emoji/15.0/
+    *   ReadMe.txt
+    *   emoji-sequences.txt
+    *   emoji-test.txt
+    *   emoji-zwj-sequences.txt 
```

Comment:
Comment from @rickmcgowan via email:

I'd suggest adding a note to the effect that the above files are *part of the UCD*, so vetting and posting needs to be done in coordination with the release of the standard. At this point Ken Whistler and the UTC release management team are in charge of that.
The other emoji data files go with the versioned *Emoji* release. And the charts are sort of evergreen and can be changed between releases.

----
My partial response: Whenever we "copy to Public" we really should copy to a staging area, and someone (currently Ken or Rick) then actually copies to Public. We can make this explicit everywhere, or just remember that intermediate step.